README.md

```diff
@@ -559,6 +559,8 @@ See also [Foreign Function Interface](https://doc.rust-lang.org/book/first-editi
   * [rust-lang/rust-bindgen](https://github.com/rust-lang/rust-bindgen) — A Rust bindings generator
 * Erlang
   * [rusterlium/rustler](https://github.com/rusterlium/rustler) — safe Rust bridge for creating Erlang NIF functions [![build badge](https://api.travis-ci.org/rusterlium/rustler.svg?branch=master)](https://travis-ci.org/rusterlium/rustler)
+* Haskell
```

Comment:
This change should not be part of this PR. If you needed to add this, create a new PR